content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,161 @@
+
+Monal continues on fighting the iOS front: From the [beta](https://monal.im/blog/4-6-betas/) upcoming are support for geo location messages, [TLS ALPN support](https://github.com/anurodhp/Monal/issues/206) and improvements on [OpenFire push notifications](https://github.com/anurodhp/Monal/issues/354) as well as many fixes with smack.
+
+
+### Servers
+
+
+
+
+### Clients and applications
+
+The maintainer of libstrophe, [Pasis](https://github.com/pasis), and contributor to Profanity, created a tool called [xmppconsole](https://github.com/pasis/xmppconsole) which sends raw XMPP stanzas over an XMPP connection and displays the XMPP stream. Main purpose is to study XEPs and debug servers implementation.
+The tool is under development, the finished version will support both GTK UI and ncurses UI. In the blog they also explained how to easier contribute commits to their repository, [read on](https://profanity-im.github.io/blog/post/contributing-a-patch-via-github/)!
+
+### Other
+
+
+## Services
+
+yax.im updated their servers from prosody 0.11 to [prosody-trunk](https://hg.prosody.im/trunk/file/d0e6d5bc7ea2/CHANGES). Read more about the changes [here](https://yaxim.org/blog/2020/04/23/new-prosody-on-yax-dot-im/).
+
+On April 25th, [observe.jabber.network](https://observe.jabber.network) was
+launched into public beta. It offers free monitoring-as-a-service for federated
+XMPP domains, checking connectivity via c2s and s2s, as well as optionally
+in-band registration and federated XMPP pings. Application details and more
+information are available on the website.
+
+
+## Extensions and specifications
+
+
+### Deferred
+
+### Updated
+
+- Version 0.4 of XEP-0333 (Chat Markers) has been released.
+Abstract:
+This specification describes a solution of marking the last received,
+displayed and acknowledged message in a chat.
+Changelog:
+Add notes about usage within MUCs. (mw)
+URL: https://xmpp.org/extensions/xep-0333.html
+
+- Version 0.3.0 of XEP-0389 (Extensible In-Band Registration) has been
+released.
+Abstract:
+This specification defines an XMPP protocol extension for in-band
+registration with instant messaging servers and other services with
+which an XMPP entity may initiate a stream. It aims to improve upon
+the state of the art and replace XEP-0077: In-Band Registration by
+allowing multi-factor registration mechanisms, and account recovery.
+Changelog:
+* Move back to experimental.
+* Add IQ variant of registration/recovery.
+* Allow client to select between multiple flows. (ssw)
+URL: https://xmpp.org/extensions/xep-0389.html
+
+Note: The information in the XEP list at https://xmpp.org/extensions/
+is updated by a separate automated process and may be stale at the
+time this email is sent. The XEP documents linked herein are up-to-
+date.
+
+### Miscellaneous
+
+**ProtoXEP**
+
+**Experimental**
+
+**Active**
+
+**Draft**
+
```

Comment:
- Version 1.18.0 of XEP-0060 (Publish-Subscribe) has been released.
Abstract: This specification defines an XMPP protocol extension for generic publish-subscribe functionality. The protocol enables XMPP entities to create nodes (topics) at a pubsub service and publish information at those nodes; an event notification (with or without payload) is then
broadcasted to all entities that have subscribed to the node. Pubsub therefore adheres to the classic Observer design pattern and can serve as the foundation for a wide variety of applications, including news feeds, content syndication, rich presence, geolocation, workflow systems, network management systems, and any other application that requires event notifications.
Changelog: Properly specifiy that an empty <item/> is invalid on publish. (jsc)
URL: https://xmpp.org/extensions/xep-0060.html

- Version 1.0.0 of XEP-0402 (PEP Native Bookmarks) has been released.
Abstract: This specification defines a syntax and storage profile for keeping a
list of chatroom bookmarks on the server.
Changelog: Advanced to Draft per Council vote from 2020-03-04. (XEP Editor (jsc))
URL: https://xmpp.org/extensions/xep-0402.html

Reply:
@Echolon after yesterday’s clarification, "Version 1.18.0 of XEP-0060 (Publish-Subscribe) has been released" should be listed in the **Updated** section.